.gitignore

```diff
@@ -25,4 +25,21 @@ libtool
 isa-l.h
 /libisal.la
 libisal.pc
+crc/crc16_t10dif_copy_perf
```

Comment:
You could change this to crc/*_perf. Same for other folders.
Also, could you include the *_test files? Thanks!

Reply:
Fixed, thank you!